docs/io.cozy.contacts.md

```diff
@@ -53,7 +54,13 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
 
 - `trashed`: {boolean} is `true` if the contact is marked for removal and will be deleted soon (e.g. after remote deletion is confirmed)
 
--   `me`: {boolean} whether the contact matches the cozy owner (defaults to `false`)
+- `me`: {boolean} whether the contact matches the cozy owner (defaults to `false`)
+
+- `metadata`: {object} previous metadata information. **Won't be updated. Don't use this!**
```

Comment:
idk if it won't be updated though? In my mind we kept it to store metadata about the contact, as opposed to metadata about the document (`metadata` vs `cozyMetadata`).

Reply:
ok! :-)